sympy/functions/elementary/integers.py

```diff
@@ -68,6 +68,8 @@ def eval(cls, arg):
             return ipart
         elif spart.is_imaginary or (S.ImaginaryUnit*spart).is_real:
             return ipart + cls(im(spart), evaluate=False)*S.ImaginaryUnit
+        elif isinstance(spart,cls):
```

Comment:
Please add spacing around commas.
`elif isinstance(spart, cls):`

Reply:
Please check the changes.